public/js/leaderboard.js

```diff
@@ -1,19 +1,99 @@
 /* set home as the active navbar element */
 $('#nav-leaderboard').addClass('active');
 
+var boardType = leaderboardTypes.OVERALL;
+var leaderboardRow;
+var leaderboardTable;
+var studentLeaderList;
+var podiumScore;
+
 /* Get HTML for the complete leaderboard table from the server and display it. */
 var fetchLeaderboard = function() {
+    
     $.ajax({
         type: 'GET',
         url: '/leaderboard-table',
         data: {
-            fullTable: true,
-            longTable: true
+            smallBoard: false
         },
         success: function(data) {
-            $('.leaderboard-content').html(data);
+            setPodiumImages();
```

Comment:
this should happen when we are setting up the table

Reply:
Fixed